patternfly/patternfly-elements
# fix: Content set hold rendering until upgrade, prefixed attribute support for overflow property in cards

##  1.3.2 patch

 - Content set cascading attributes patch from issue found on dotcom
 - Added back fallback styles for pfe-overflow prefixed attributes (not a property on host so needs to be backwards compatible on the scss level)

### Preview

Link(s) to demo page(s) where this element can be viewed:
- [Content-set demo](https://deploy-preview-1427--happy-galileo-ea79c4.netlify.app/elements/pfe-content-set/demo#attributes) 
- [Card demo](https://deploy-preview-1427--happy-galileo-ea79c4.netlify.app/elements/pfe-card/demo) 
- (If you have Red Hat VPN access): https://web-dev-drupal-carobert.dev.a1.vary.redhat.com/en/demo-accordions-and-tabs-pattern

### Testing instructions

1. Check that the attributes on the demo page cascade to the rendered component: [demo page](https://deploy-preview-1427--happy-galileo-ea79c4.netlify.app/elements/pfe-content-set/demo#attributes)
    - vertical (tabs)
    - selected-index (tabs)
    - align (tabs)
    - variant (tabs)
    - tab-history (tabs)
    - disclosure (accordion)
2. Check that the overflow images fill the card: [Card demo](https://deploy-preview-1427--happy-galileo-ea79c4.netlify.app/elements/pfe-card/demo) 


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist





### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
